beesdoo_shift/models/res_partner.py

```diff
@@ -229,55 +229,58 @@ def get_next_shifts(self):
             task_template = self.env["beesdoo.shift.template"].search(
                 [("worker_ids", "in", self.id)], limit=1
             )
-            main_shift = self.env["beesdoo.shift.shift"].search(
-                [
-                    ("task_template_id", "=", task_template[0].id),
-                    ("start_time", "!=", False),
-                    ("end_time", "!=", False),
-                ],
-                order="start_time desc",
-                limit=1,
-            )
+            if task_template:
```

Comment:
Instead of indenting everything with `if task_template`, could we do something like:

```python
task_template = [...]
if not task_template:
    return generated_shifts, []
main_shift = [...]
```

Because: Flat is better than nested.

Reply:
Done